app/views/layouts/application.html.erb

```diff
@@ -40,6 +40,12 @@
       <%= link_to t('navigation.projects'), projects_path %>
     </li>
   <% end %>
+
+  <% if user_can_administer_taxonomy? %>
```

Comment:
If we ever decide to open up this role to other users, we'll need to create a separate check for it, but at the moment it's fine to put it with the taxonomy permissions.

Reply:
I would be happy to create an explicit method (eg user_can_administer_facet_groups?). 
I saw it as a trade of between DRY and Clean (meaningful names and functions) code. Very interested in your views. Would it be considered repeating?

Reply:
If we were going to really DRY-up the code, we'd just make `gds_editor?` public and use that instead, but that doesn't really tell you what that user can do.

I would have created a new method, just to make it clear that administering the taxonomy and administering facet groups are two different things, even though at the moment, you need the same permissions for both. But that's because my own preference is for code that's easy to read so you don't have to do too much digging to find out what something means.

Reply:
Thank you! Really insightful. I will created a new method.

Reply:
It looks good! 💯  There was only one nitpicky comment.